csrc/python_frontend/fusion_state.cpp

```diff
@@ -65,14 +89,24 @@ void FusionState::resetFusionState(Fusion* fusion, size_t size) {
   TORCH_CHECK(fusion != nullptr, "Fusion is undefined.");
   fusion_ = fusion;
   fusion_state_.clear();
-  fusion_state_.resize(size, nullptr);
+  fusion_state_.resize(size, {});
 }
 
 void FusionState::addFusionState(Val* val) {
+  fusion_state_.push_back({val});
+}
+
+void FusionState::addFusionState(std::vector<Val*> val) {
```

Comment:
Should this be renamed as `getFusionStateVector`?

Reply:
I originally wanted the `Vector` support to be symmetric with `Tensor` and  `Scalar`.  However, given there is no IR support for `Vector`s, it does make it programmatically clear-er to name all the methods that support `Vector` with "Vector".